flutter_local_notifications/lib/src/platform_specifics/ios/notification_details.dart

```diff
@@ -71,4 +72,38 @@ class IOSNotificationDetails {
   ///
   /// This property is only applicable to iOS 10 or newer.
   final String? threadIdentifier;
+
+  /// The interruption level that indicates the priority and
+  /// delivery timing of a notification.
+  ///
+  /// This property is only applicable to iOS 10 or newer.
```

Comment:
The comment says applicable for iOS 10 or newer but shouldn't that be iOS 15 or newer?

Reply:
Correct. I wil fix that.